Update repository installation status message (#7148)
EnterpriseDB/docs: product_docs/docs/pgpool/4/installing/linux_x86_64/pgpool_other_linux_9.mdx

  `dnf repolist | grep enterprisedb`

>   If no output is generated, the repository is not installed. Then you need to setup EDB repository.

Reviewer: ```suggestion
 If no output is generated, the EDB repository is not installed. To install it, set up the EDB repository.
```
